### PR TITLE
Crypto finance 

### PR DESCRIPTION
**Pull request by dialite**
This mobile web application project has the following features:

Capstone Project- Crypto Finance Metrics

- A single-page application (SPA) built with React and Redux.
- The data retrieved from the API is stored in the Redux store.
- Obtain and display crypto data from CoinStats.API.

- Set up a search bar to be able to filter based on the company's names.

- Create tests using React Testing Library.
- Deployed the project and added the live demo link in the README.md file.

### Quick links
[Live Demo](https://63b760648bdb802e810a0063--calm-chebakia-029401.netlify.app/)
[Project presentation](https://www.loom.com/share/a3ef9ab9179d4d318e08513773c767f6)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,177 @@
-# Financial-metrics
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
```
Comment (rloterh) [inline README.md:5]: - [ ] OPTIONAL
Your logo is broken. Kindly ensure to fix it.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Raymond, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project, but you are almost there! 🚦 🚦   🏃 

## Highlights 👍  
- Nice presentation of this project
- Good job acknowledging Nelson Sakwa on Behance for the [Original design idea](https://www.behance.net/sakwadesignstudio).
- API fetch properly implemented
- State management with Redux properly implemented
- Navigation working as intended
- Properly implemented unit and integration test 
- Filter works great! 💪 
- Good ReadMe

## Required Changes ♻️
_Kindly check the inline comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

_Kindly check the inline comments marked **[OPTIONAL]** under the review._

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by dialite** [comment]
Thank you for the feedback review. All observation duly implemented.

**Review by mwanawabangona — APPROVED** [review]
Hi @dialite

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implementing the requested changes !! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**